test/sql/inet/test_inet_functions.test

```diff
@@ -45,3 +50,8 @@ SELECT INET '127.0.0.31' - -32;
 statement error
 SELECT INET '0.0.0.0' - 32;
 ----
+
+# overflow
+statement error
+select INET '255.255.255.255' + 1
+----
```

Comment:
Could you add an expected error message here?

Reply:
Makes sense, I'll work on adding these cases.

Reply:
Added expected outputs to various error tests.